### PR TITLE
Sql

### PR DESCRIPTION
**Pull request by johnnycalderon86**
Database name: Notes_DB
Table: notes_table
I have 3 columns: "AuthorName", "Title", and "Notes"

Both AuthorName and Title have a limitation of 100 characters. This is to stop the user from typing too long of a text in these fields. They also have a value of NOT NULL  so that the field must have a value to insert a new record.

Notes have a character limitation of 500 characters, which should be enough to write a note. It also has the value NOT NULL.

### CODE REVIEW

**Review by johnnycalderon86 — COMMENTED** [review]
p

File: Mysql.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE Notes_DB;
+CREATE TABLE notes_table (
```
Comment (SeppeVerhavert) [inline Mysql.sql:2]: Why is notes_table without capitals?

File: Mysql.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE Notes_DB;
+CREATE TABLE notes_table ( 
+     AuthorName VARCHAR(100) NOT NULL , 
+     Title VARCHAR(100) NOT NULL ,
```
Comment (SeppeVerhavert) [inline Mysql.sql]: Why is authorname and title allowed to be 100 characters long?

File: Mysql.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+      Note VARCHAR(500) NOT NULL ,
+)
+
```
Comment (SeppeVerhavert) [inline Mysql.sql]: Why do you not add a primary key?

**Review by SeppeVerhavert — COMMENTED** [review]
Good code, looks clean.

File: Mysql.sql (hunk 3 above)
Comment (ghost) [reply Mysql.sql]: like seppe said you need to put a private key on the title then there can only be one of the same title. if you want more then one title with the same name you should make a column named id and give that a primary key.

**Comment by bennyho94** [comment]
Despite the already mentioned feedback of my colleagues, I don't have much to add besides that your code is understandable and clean.

**Pull request merged**